packages/beacon-node/src/chain/beaconProposerCache.ts

```diff
@@ -7,19 +7,28 @@ const PROPOSER_PRESERVE_EPOCHS = 2;
 
 export type ProposerPreparationData = routes.validator.ProposerPreparationData;
 
+/** Cache to track the validators registored with beacon node. Could eventually be promoted as a
```

Comment:
` Could eventually be promoted as a full monitor` is not clear to me, what is remaining for monitoring block proposals?

Reply:
tbh i am not sure, monitoring can be for other things as well, but this class holds the registered validators so can naturally be become a validator monitor

Reply:
removed the validatorMonitor flag complications for future PRs